CHANGES_NEXT_RELEASE

```diff
@@ -1,3 +1,4 @@
-- [cygnus-ngsi][feature] Add truncation options based on time and size (#1317)
+- [cygnus-ngsi][feature] Add global data truncation options based on time and size (#1317)
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1351/commits/733214a6c45b3629c8054f3043d452340cf06c84